ansible/roles/ocp4-workload-ccnrd/tasks/install-mta.yaml

```diff
@@ -1,31 +1,31 @@
 ---
-# deploy rhamt
-- name: Deploy RHAMT
+# deploy mta
+- name: Deploy MTA
   shell: oc process -f - | oc create -n labs-infra  -f -
   args:
-    stdin: "{{ lookup('file', './files/rhamt-template.yaml') }}"
+    stdin: "{{ lookup('file', './files/mta-template.yaml') }}"
 
-- name: TODO - scale RHAMT executor to quarter the number of users
+- name: scale MTA executor to quarter the number of users
   shell: |
-         oc scale dc/rhamt-web-console-executor --replicas={{ ((num_users|int) / 3) | int }} -n labs-infra
+         oc scale dc/mta-web-console-executor --replicas={{ ((num_users|int) / 3) | int }} -n labs-infra
```

Comment:
Please convert this to use `k8s` module. Something as simple as this should be sufficient.

```
k8s:
  state: present
  apply: yes
  definition:
    apiVersion: apps.openshift.io/v1
    kind: DeploymentConfig
    metadata:
      name: mta-web-console-executor
      namespace: labs-infra
    spec:
      replicas: {{ ((num_users|int) / 3) | int }}
```